rules/community/okta/okta_mfa_pw_reset_in_short_window.yaral

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+rule okta_mfa_pw_reset {
+
+  meta:
+    author = "Drew Pilarski - Tempus AI"
+    description = "This detection is looking for MFA and Password resets in a 30 minute window. This is suspicious given normal activity should only constitute a password reset OR MFA reset (ala a replacement phone)"
+    notes = "https://developer.okta.com/docs/reference/api/event-types/"
```

Comment:
```suggestion
    reference = "https://developer.okta.com/docs/reference/api/event-types/"
```

I'm going to suggest some changes like this so that the rule aligns with the [style guide](https://github.com/chronicle/detection-rules/blob/main/STYLE_GUIDE.md).